README.md

```diff
@@ -61,10 +61,12 @@ When necessary, client-zip will generate Zip64 archives. Those are not readable
 
 # Usage
 
-The module exports two functions:
+The module exports three functions:
 ```typescript
 function downloadZip(files: ForAwaitable<InputTypes>, options?: Options): Response
 
+function makeZip(files: ForAwaitable<InputTypes>): ReadableStream
+
```

Comment:
It seems that the stream on the Response might not be compatible with other streams. At least, I didn't find a way how to pipe it to another place where I needed it (it complained it is not a stream 🤷).

Reply:
Its better anyway to use it directly than wrap it and then unwrap it again, also it has extra parameters which are not needed at all for simple stream. So I think it does no harm at all and makes the library more flexible.

Reply:
And it works if you use the output of `makeZip` directly ? considering it's the same stream used to make the Response, that'd be weird.

Reply:
Would you mind sharing the exact error you get when you try to use `response.body` and, if possible, the code leading to that error ?

Reply:
The error says: `TypeError: Failed to execute 'pipeTo' on 'ReadableStream': parameter 1 is not of type 'WritableStream'.`

Example of code:

```
let { readable, writable } = new TransformStream();
const response = downloadZip(...);
response.body?.pipeTo(writable);
```

It has the same issue with WritableStream directly. In both cases, when using the stream directly, all is fine.

Reply:
yes, sorry, I meant WritableStream.

Reply:
I don't understand why it doesn't work. Either way, I don't think it's bad to provide stream directly without wrapping it in case its not needed anyway, or is it?